### PR TITLE
[expo-camera] Web detection of inverted QR codes

### LINKED ISSUE #16103: Expo-Camera (web): QR code not recognized with inverted colors
### Summary

A QR code with a black foreground, white background is recognized. However, a white foreground and black background is not recognized. Possibly already fixed for BarCodeScanner, but not applied to Camera/Web (#1736).

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

Web

### SDK Version (managed workflow only)

_No response_

### Environment

Expo CLI 4.12.10 environment info:
    System:
      OS: macOS 12.0.1
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.18.2 - /usr/local/bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 6.14.5 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.0.1, iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0
      Android SDK:
        API Levels: 28, 29, 30
        Build Tools: 28.0.3, 29.0.2, 30.0.0, 30.0.2
        System Images: android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
    IDEs:
      Android Studio: 4.1 AI-201.8743.12.41.7199119
      Xcode: 13.1/13A1030d - /usr/bin/xcodebuild
    npmPackages:
      expo: ^43.0.0 => 43.0.1 
      react: 17.0.1 => 17.0.1 
      react-dom: 17.0.1 => 17.0.1 
      react-native: 0.64.2 => 0.64.2 
      react-native-web: 0.17.1 => 0.17.1 
    npmGlobalPackages:
      expo-cli: 4.12.10
    Expo Workflow: bare

### Reproducible demo

I opened the Snack below on my computer and went to the Web tab (I needed to hit the [expand/maximize](https://snack.expo.dev/dist/assets/d93a5d445d7b7a8be13f38362a9dcf7e.png) button to get Web to work). Then I opened the below QR codes on my phone and switched between showing them to my computer's camera.

https://snack.expo.dev/@claypot/62b702

[Here are two identical QR codes for https://expo.dev/](https://imgur.com/a/cRl0oSr). The only difference is inverting the foreground and background colors. White codes on black backgrounds were not registered, while the black code on white background is immediately detected.

Issue #1736: It seems like it was fixed for BarCodeScanner, but not the Camera module, which is required for web (#15442).

### PR DESCRIPTION
**Pull request by rissois**
NOTE: Second pull request ever, please double-check.

# Why

Web was previously unable to detect light foregrounds and dark backgrounds for QR codes

# How

Changed options argument sent to jsqr allowing for inversionAttempts.

# Test Plan

Used patch-package... this is about as much information as I can provide

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [?] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [?] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by rissois** [comment]
@expo-bot I know you are a bot, but I do not know how to do what you are asking me to do.

**Comment by Simek** [comment]
> [...] I do not know how to do what you are asking me to do.

Hi @rissois, you just need to add an entry (sentence) to this file in "Unreleased" section, probably in `New feature` part:
* https://github.com/expo/expo/blob/main/packages/expo-camera/CHANGELOG.md

Fixes #16103

**Comment by rissois** [comment]
Thank you @Simek , your step-by-step instructions were very helpful!

File: packages/expo-camera/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- Updated `useWebQRScanner` to allow for `inversionAttempts` of QR codes with light foregrounds and dark backgrounds ([#16106](https://github.com/expo/expo/pull/16106) by [@rissois](https://github.com/rissois))
```
Comment (Simek) [inline packages/expo-camera/CHANGELOG.md:9]:
```suggestion
- Update `useWebQRScanner` to allow scanning QR codes with inverted colors (light foreground and dark background). ([#16106](https://github.com/expo/expo/pull/16106) by [@rissois](https://github.com/rissois))
```

File: yarn.lock [hunk 2]
```diff
@@ -8548,6 +8548,17 @@ expo-asset-utils@~3.0.0:
...
+  version "0.5.5"
+  resolved "https://registry.yarnpkg.com/expo-modules-autolinking/-/expo-modules-autolinking-0.5.5.tgz#6bcc42072dcbdfca79d207b7f549f1fdb54a2b74"
+  integrity sha512-bILEG0Fg+ZhIhdEaShHzsEN1WC0hUmXJ5Kcd4cd+8rVk1Ead9vRZxA/yLx1cNBDCOwMe0GAMrhF7TKT+A1P+YA==
+  dependencies:
+    chalk "^4.1.0"
+    commander "^7.2.0"
+    fast-glob "^3.2.5"
+    find-up "^5.0.0"
+    fs-extra "^9.1.0"
+
```
Comment (Simek) [inline yarn.lock:8561]:
```suggestion
```

Not sure why lock has been updated here, let's revert it.

**Review by Simek — APPROVED** [review]
LGTM, thank you for figuring this option out and preparing the PR! 👍

**Pull request merged**